pkg/skaffold/deploy/resource/pod.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import (
+	"context"
+	"time"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/runcontext"
+	"github.com/pkg/errors"
+)
+
+const (
+	PodType     = "pod"
```

Comment:
nit: we have this defined in [constants.go](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/constants/constants.go#L76)

Reply:
i like package level constants then globally defined for reasons:
- they are defined closer to where their usage is. (especially if they are not across multiple packages)

I know in skaffold they are all defined in one file.

I was trying to find what is the best recommendation is and could not get to an answer.

Reply:
sure, although this constant would be used across multiple packages (used in port forwarding as well)